packages/node_modules/@webex/webex-core/src/lib/services/service-url.js

```diff
@@ -44,19 +44,24 @@ const ServiceUrl = AmpState.extend({
   },
 
   /**
-   * Get the current host url with the highest
-   * priority.
-   * @returns {string}
+   * Get the current host url with the highest priority.
+   *
+   * @param {string} [clusterId] - The cluster to retrieve the priority host from.
```

Comment:
Add what cluster it uses if the method parameter isn't provided (home cluster).

Reply:
Will append this JSDoc note.